app/crowdSourceData/aashishdhakal.json

```diff
@@ -0,0 +1,15 @@
+name: "Aashish Dhakal",
```

Comment:
this is not a valid json file. @ujwaldhakal may be you should add a check in some ci pipeline to check if the file being used is an actual json file.

Reply:
ok :)